styles.css

```diff
@@ -186,22 +186,57 @@ input {
 
 /*** CURRENT ACTIVITY ***/
 
-/********************/
+/* --------------------------------------------------->> */
 .timer-page {
-  border: 2px solid red;
-  display: inline;
+  /* border: 2px solid red; */
+  display: flex;
   flex-direction: column;
-  justify-content: center;
-  height: 90%;
-  width: 80%;
+  justify-content: space-around;
+  align-items: center;
+  height: 70%;
+  width: 40%;
 }
 
 .time {
-  display: flex;
+  color: #CBC9CF;
+  font-size: 4.5em;
+  font-weight: bold;
+}
+
+.timer-activity-description {
+  font-size: 1.3em;
+  color: #CBC9CF;
+}
+
```

Comment:
Does the timer circle outline need to be different colors depending on the activity category passed in?